include/package-pack.mk

```diff
@@ -329,6 +329,10 @@ else
 		rm -rf $$(IDIR_$(1))/CONTROL/conffiles; \
 	fi
 
+    ifneq ($$(CONFIG_IPK_FILES_CHECKSUMS),)
+	if [ -f $$(IDIR_$(1))/CONTROL/files-sha256sum ]; then mv -f $$(IDIR_$(1))/CONTROL/files-sha256sum $$(IDIR_$(1))/lib/apk/packages/$(1).files-sha256sum; fi
```

Comment:
Isn't that effectively the same thing already provided as built-in feature of APK?
Afaik `apk audit` is exactly for that and we don't need additional files with hashes.

Reply:
Yea I think you can drop this commit entirely... sorry.

Reply:
Ok I will change this commit to depend on !USE_APK